.gitignore

```diff
@@ -250,3 +250,16 @@ GSYMS
 GPATH
 tags
 TAGS
+
+# Files generated my cmake
+CMakeLists.txt.user
+CMakeCache.txt
+CMakeFiles
+CMakeScripts
+Testing
+Makefile
+cmake_install.cmake
+install_manifest.txt
+compile_commands.json
+CTestTestfile.cmake
+_deps
```

Comment:
We already ignore `*.user` on line 169, and `compile_commands.json` on line 115.

Reply:
Have removed those from this list.